odoo-11.0/addons_custom/payslip/data/button_display_payslip.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_employee_form_inherit" model="ir.ui.view">
+        <field name="name">hr.employee.form.inherit</field>
+        <field name="model">hr.employee</field>
+        <field name="inherit_id" ref="hr.view_employee_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//button[@name='toggle_active']" position="after">
+                <button name="action_view_payslips" type="object"
+                        class="oe_stat_button" string="View Payslip"
+                        icon="fa-file-text-o"
+                        attrs="{'invisible': [('id', '=', False)]}"
+                        context="{'active_id': active_id}">
+                </button>
+            </xpath>
+            <xpath expr="//field[@name='job_id']" position="after">
+                <!-- Add other fields here -->
+                <field name="status"/>
+            </xpath>
+            <xpath expr="//field[@name='department_id']" position="before">
+                <field name="date_onboard" readonly="1" attrs="{'invisible': ['&amp;', ('status', '!=', 'draft'), ('status', '!=', 'waiting_approve'), ('status', '!=', 'approved')]}"/>
+                <field name="date_terminated" readonly="1" attrs="{'invisible': [('status', '!=', 'terminated')]}" />
+            </xpath>
+            <xpath expr="//field[@name='notes']" position="after">
+                <!-- Thêm action button submit -->
+                <button name="action_submit" string="Submit" type="object" attrs="{'invisible': [('status', 'not in', ['draft'])]}" class="oe_highlight"/>
```

Comment:
khi nào thì nên dùng not in em nhỉ?

Reply:
dùng not in khi muốn kiểm tra xem giá trị có nằm trong danh sách giá trị hay không, còn != là khi so sánh với 1 giá trị, trong th này dùng được cả 2 cách nhưng chắc là chỉ cần dùng !=